src/Algebra/Module/Properties.agda

```diff
@@ -18,7 +18,8 @@ module Algebra.Module.Properties
 
 open Module mod
 open import Algebra.Module.Properties.Semimodule semimodule public
-open import Algebra.Properties.Group using (⁻¹-involutive)
-
--ᴹ-involutive : Involutive _≈ᴹ_ -ᴹ_
--ᴹ-involutive = ⁻¹-involutive +ᴹ-group
+open import Algebra.Module.Properties.LeftModule leftModule public
+  using (identityˡ-uniqueᴹ; identityʳ-uniqueᴹ)
```

Comment:
This API change isn't listed in the CHANGELOG?

Reply:
Good catch! Thanks.